plasmapy/particles/ionization_state_collection.py

```diff
@@ -79,7 +83,7 @@ class IonizationStates:
     {'H': array([0.5, 0.5]), 'He': array([0.95, 0.05, 0.  ])}
 
     The number densities are given by the ionic fractions multiplied by
-    the abundance and the
+    the abundance and the number density scaling factor ``n``.
```

Comment:
This parameter was originally the number density of hydrogen, but this would have required that hydrogen was one of the elements included in the collection.  The number density of an ionization state is: (abundance of element) × (ionic fraction) × (number density scaling factor).  It's not the overall density but rather a multiplicative factor with units of number density.  I'll try to clarify that in the docstring.  I'm wondering if it'd be better to rename ``n`` as ``n0`` or something.

Reply:
Is the scaling factor more easily available at input time than the number density? It might be better to calculate it ourselves from number density, given that we already grab the abundances.

Reply:
Thanks!  I'll change `n` to `n0`.